docs/kb/integrations/defect-dojo-integration.md

```diff
@@ -126,6 +126,17 @@ In the example, the values are:
 * `PRODUCT` = chess-game
 * `IMPORT_SEMGREP_TO_DEFECTDOJO_SHA_CHECKSUM` = c41aed4055adeee415b795cc17a069b144fb51bc31f6c4925be3b82d0b54de33 Uimport_semgrep_to_defect_dojo.py
 
+The content for this last variable was generated with the following command:
+`shasum -a 256 -U import_semgrep_to_defect_dojo.py`
+it will generate an unique key taking as input the content of the script and it will be used to verify the integrity. 
```

Comment:
```suggestion
This command generates a unique hash, taking as input the content of the script, and it will be used to verify that the script has not changed. 
```